tests/LaunchDarkly.ClientSdk.Tests/Internal/DataSources/DataSourceUpdateSinkImplTest.cs

```diff
@@ -13,8 +13,9 @@ public class DataSourceUpdateSinkImplTest : BaseTest
         private readonly FlagDataManager _store;
         private readonly FlagTrackerImpl _flagTracker;
         private readonly DataSourceUpdateSinkImpl _updateSink;
-        private readonly Context _basicUser = Context.New("user-key");
-        private readonly Context _otherUser = Context.New("other-key");
+        
+        private readonly Context _basicUser = Context.NewMulti(Context.New(ContextKind.Of("user"), "user-key1"), Context.New(ContextKind.Of("custom-kind"), "custom-key1"));
+        private readonly Context _otherUser = Context.NewMulti(Context.New(ContextKind.Of("user"), "user-key2"), Context.New(ContextKind.Of("custom-kind"), "custom-key2"));
```

Comment:
For reviewers: Using multi-contexts in the test here causes the tests to fail without the fix.